tests/test_miscellaneous.py

```diff
@@ -44,3 +46,24 @@ def test_properties_setters(self):
     def test_asdict(self):
         dict_ = self.subject._asdict()
         assert_equal(dict_['script'], self.subject.script)
+
+
+# This does not subclass TestCase so test generators work
```

Comment:
You don't need to add more comments to the code, but I woiuld like to see more details about these test generators.